scala/Scala.g4

```diff
@@ -186,10 +186,15 @@ exprs
    ;
 
 argumentExprs
-   : '(' exprs? ')'
-   | '(' (exprs ',')? postfixExpr ':' '_' '*' ')'
-   | NL? blockExpr
-   ;
+    : '(' args ')'
+    | '{' args '}'
+    | NL? blockExpr
+    ;
+
+args
+    : exprs?
+    | (exprs ',')? postfixExpr ':' '_' '*' // should be (':' | '_' | '*') ?
```

Comment:
If you know the correct syntax, please update it and remove this comment. Or add a TODO here, so someone can fix it later.

Reply:
Hi @studentmain ,

I have did the correction. Can you verify & close the PR.